disnake/user.py

```diff
@@ -71,12 +73,17 @@ def __init__(
 
     def __repr__(self) -> str:
         return (
-            f"<BaseUser id={self.id} name={self.name!r} discriminator={self.discriminator!r}"
-            f" bot={self.bot} system={self.system}>"
+            f"<BaseUser id={self.id} name={self.name!r} global_name={self.global_name!r}"
+            f" discriminator={self.discriminator!r} bot={self.bot} system={self.system}>"
         )
 
     def __str__(self) -> str:
-        return f"{self.name}#{self.discriminator}"
+        discriminator = self.discriminator
+        if discriminator == "0":
+            # TODO: add "@" prefix?
```

Comment:
afaik usernames will be displayed without the "@" prefix

Reply:
yeah, looks like Discord changed their mind about how to display this. I'll remove the comment, ty!